src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,79 @@
+package duke.tasks;
+
+import duke.storage.Storage;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskList {
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+
+    static String countTasks() {
+        String pluralSuffix = taskList.size() == 1 ? "" : "s";
+        return String.format("%s task%s", taskList.size(), pluralSuffix);
+    }
+
+    static String add(Task task) {
+        taskList.add(task);
+
+        return String.format("Great, have added the following task for you:\n%s\nNow you have %s.", task, countTasks());
+    }
+
+    static Task get(int index) {
+        return taskList.get(index - 1);
+    }
+
+    static String list() {
+        int task_no = 1;
+        StringBuilder prompt = new StringBuilder();
+
+        prompt.append(String.format("You have %s in the list\n", countTasks()));
+
+        for (Task t : taskList) {
+            prompt.append(String.format("%d. %s\n", task_no++, t.toString()));
+        }
```

Comment:
I like using enhanced for-loops and `forEach` to iterate through my Collections as well, but since you have to a variable perhaps consider the _boring_ for-loop? 😆 

```suggestion
        for (int i = 0; i < taskList.size(); i++) {
            prompt.append(String.format("%d. %s\n", i+1, t.toString()));
        }
```

At least this way you can get rid of line 29